search/everywhere_search_callback.hpp

```diff
@@ -7,6 +7,11 @@
 
 namespace search
 {
+struct ProductInfo
+{
+  bool m_isLocalAdsCustomer = false;
+  float m_ugcRating = -1;
```

Comment:
optional: use boost::optional to represent invalid values, or a special const, say, kInvalidUGCRating.